src/task/task.js

```diff
@@ -0,0 +1,51 @@
+const randomId = require("../utils/randomId");
+
+const getTasks = async (apiClient, { projectId }) => {
+  try {
+    const {
+      data: {
+        changes: {
+          tasks: { updated },
+        },
+      },
+    } = await apiClient.get("sync", {
+      collection_name: "tasks",
+    });
+
+    if (projectId) {
+      return updated.filter((task) => task.project_id === projectId);
+    }
+
+    return updated;
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const addTask = async (apiClient, { taskName, projectId }) => {
+  try {
+    await apiClient.post("sync", {
+      tasks: {
+        created: [
+          {
+            id: randomId(),
+            name: taskName,
+            is_all_day: true,
```

Comment:
hm, is this necessary for it to work? In theory, this should only be necessary if due_at !== null, but perhaps backend sanitization is broken

Reply:
This, tbh, I don't know. Basically that was the first thing I've tried and it worked. I'll check if it is necessary.